starknet_py/net/signer/stark_curve_signer.py

```diff
@@ -73,13 +73,13 @@ def _sign_transaction(self, transaction: Invoke):
             version=transaction.version,
             contract_address=self.address,
             entry_point_selector=DEFAULT_ENTRY_POINT_SELECTOR
-            if transaction.version == 1
+            if transaction.version != 0
```

Comment:
What if someone wants to sign a transaction whit `version = 2**128`? It should be created exactly like the `0` version